docs/Measuring_Resonances.md

```diff
@@ -65,36 +65,28 @@ the system that may damage the electronics.
 
 ## Software installation
 
-Note that resonance measurements and shaper auto-calibration require additional
-software dependencies not installed by default. First, you will have to run on
-your Raspberry Pi the following command:
+Because setting up the software for auto-calibration takes significant
+time and resources, it is not installed by default. Run the following
+script to setup `numpy` and `matplotlib` to get started:
 ```
-~/klippy-env/bin/pip install -v numpy
+~/klipper/scripts/setup-shaper-calibrate.sh
 ```
-to install `numpy` package. Note that, depending on the performance of the
+Note that, depending on the performance of the
 CPU, it may take *a lot* of time, up to 10-20 minutes. Be patient and wait
 for the completion of the installation. On some occasions, if the board has
-too little RAM, the installation may fail and you will need to enable swap.
+too little RAM, the installation may fail, and you will need to
+[enable swap](https://nebl.io/neblio-university/enabling-increasing-raspberry-pi-swap/).
 
-Next, run the following command to install the additional dependencies:
-```
-sudo apt install python-numpy python-matplotlib
-```
-
-Afterwards, check and follow the instructions in the
-[RPi Microcontroller document](RPi_microcontroller.md) to setup the
-"linux mcu" on the Raspberry Pi.
-
-Make sure the Linux SPI driver is enabled by running `sudo
-raspi-config` and enabling SPI under the "Interfacing options" menu.
+If you intend to connect the ADXL345 to your Raspberry Pi, check and follow the
+instructions in the [RPi Microcontroller document](RPi_microcontroller.md) to setup the
+"linux mcu" on the Raspberry Pi. Make sure the Linux SPI driver is enabled by running
+`sudo raspi-config` and enabling SPI under the "Interfacing options" menu.
 
 Add the following to the printer.cfg file:
 ```
-[mcu rpi]
-serial: /tmp/klipper_host_mcu
-
 [adxl345]
-cs_pin: rpi:None
+cs_pin: AB0 # The CS pin on your sensor that may be connected to your
```

Comment:
The big advantage of the previous 'rpi:None' was that this part was copy-pasteable. Maybe it's indeed better to remove [mcu rpi] section, but could you please keep 'rpi:None' as the default option here? Otherwise, the correct way to define the adxl345 connected to the RPi is not defined anywhere. And that 'rpi:None' syntax is very much not obvious. The comment is fine though.

Reply:
the old documentation is now back in it's own section